dotenv-linter/README.md

```diff
@@ -20,6 +20,12 @@ On Windows you'll get an error like this:
 You need to download and install the
 [Microsoft Visual C++ Redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads).
 
+### Files
+
+```text
+~/.local/bin/dotenv-linter
+~/.local/opt/dotenv-linter-{version}
```

Comment:
I believe this also has a global config and possibly a local config file (same as I described for flutter). Please check the official documentation and update accordingly.

Reply:
Haven't found config files for dotenv-linter, just explicit skips for the command call.